### PR TITLE
Add link to Gopher Slack invite

### PR DESCRIPTION
**Pull request by breml**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -67,7 +67,7 @@ <h1>B&auml;rner Go Meetup</h1>
       </li>
       <li>
         To contact us, give feedback or just hang out, check our
-        <a href="https://gophers.slack.com/archives/C039LSKJCN9" target="_blank">slack channel</a>.
+        <a href="https://gophers.slack.com/archives/C039LSKJCN9" target="_blank">slack channel <code>#b&auml;rner-go</code></a> on Gopher Slack.<br/>Not yet hanging out there, get an <a href="https://invite.slack.golangbridge.org/">invite</a> and join.
```
Comment (pfiaux) [inline index.html]: Does the line break work with the layout or is it a bit strange in a list?

Possible alternative:
```suggestion
        <a href="https://gophers.slack.com/archives/C039LSKJCN9" target="_blank">slack channel <code>#b&auml;rner-go</code></a> on Gopher Slack (not yet hanging out there? get an <a href="https://invite.slack.golangbridge.org/">invite</a> and join).
```

**Review by pfiaux — APPROVED** [review]
Good catch, I think we forget how many people might not be in it.

That might be a good warm up question at a meetup -> how many people are in the gopher slack?

File: index.html (hunk 1 above)
Comment (breml) [reply index.html]: Funny, I had it like this first and then though, that parenthesis make the text unnecessarily hard to read (looks more like code than prose). What do the others think?

**Comment by breml** [comment]
> That might be a good warm up question at a meetup -> how many people are in the gopher slack?

@philipsahli Maybe you would like to take this question and add it to the slides as "ice breaker" at the beginning of the meetup.

**Review by cldmstr — APPROVED** [review]

File: index.html (hunk 1 above)
Comment (breml) [reply index.html]: Maybe the good compromise is to keep it as a regular sentence (without the parenthesis) but add the questionmark.

```suggestion
        <a href="https://gophers.slack.com/archives/C039LSKJCN9" target="_blank">slack channel <code>#b&auml;rner-go</code></a> on Gopher Slack.<br/>Not yet hanging out there? Get an <a href="https://invite.slack.golangbridge.org/">invite</a> and join.
```

**Comment by philipsahli** [comment]
> > That might be a good warm up question at a meetup -> how many people are in the gopher slack?
> 
> @philipsahli Maybe you would like to take this question and add it to the slides as "ice breaker" at the beginning of the meetup.

I have added a slide to ask with links as qr codes

**Pull request merged**